src/dmd/frontend.h

```diff
@@ -336,6 +337,27 @@ class ASTNode : public RootObject
     ASTNode();
 };
 
+struct ParentSymbolAttribute final
+{
+    union
+    {
+    private:
+        LinkDeclaration* link;
+        CPPNamespaceDeclaration* cpp;
+    };
+    bool isLinkDecl;
+public:
+    void accept(Visitor* v);
+    ParentSymbolAttribute& opAssign(CPPNamespaceDeclaration* cpp);
+    ParentSymbolAttribute& opAssign(LinkDeclaration* link);
```

Comment:
And so is it ? It should be `operator=`:
```
0000000100029884 T __ZN21ParentSymbolAttributeaSEP15LinkDeclaration
000000010002985c T __ZN21ParentSymbolAttributeaSEP23CPPNamespaceDeclaration
```

Reply:
Yes, operator overloads were probably not considered for the initial implementation. Probably because the spec states that those cannot be accessed from C++ IIRC.